docs/introduction-to-smart-contracts.rst

```diff
@@ -33,9 +33,9 @@ Storage
 The first line simply tells that the source code is written for
 Solidity version 0.4.0 or anything newer that does not break functionality
 (up to, but not including, version 0.5.0). This is to ensure that the
-contract does not suddenly behave differently with a new compiler version.
+contract does not suddenly behave differently with a new compiler version. For more information about pragma, see [here](https://en.wikipedia.org/wiki/Pragma_once).
 
-A contract in the sense of Solidity is a collection of code (its functions) and
+A contract in the sense of Solidity is a collection of code (its [functions](https://en.wikipedia.org/wiki/Subroutine)) and
```

Comment:
I don't think the link is necessary here.  Solidity functions are not necessarily subroutines.  Anyway, the word "function" is explained in this sentence as a collection of code.  Maybe typeset it like `its *functions*` as in the next phrase about state.